From: Michael Ihde <michael.ihde@axiosengineering.com>
Date: Sun, 17 Mar 2013 17:48:59 +0000
Subject: Fix ossie.pc so that components compile without modification

---
 src/ossie.pc.in |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/ossie.pc.in b/src/ossie.pc.in
index 2ec3658..d30fab6 100644
--- a/src/ossie.pc.in
+++ b/src/ossie.pc.in
@@ -9,5 +9,5 @@ Name: ossie
 Description: OSSIE Core Framework
 Requires:
 Version: @VERSION@
-Libs: -L${libdir} -lossieidl -lossieparser -lossiecf @LOG4CXX_LIBS@
-Cflags: -I${includedir} -I${extraincludedir} -I${idldir} @LOG4CXX_CFLAGS@ @HAVE_LOG4CXX@ @ENABLE_EVENTS_FLAGS@
+Libs: -L${libdir} -lossiecf -lossieidl -lossieparser @OMNICOS_LIBS@ @OMNIORB_LIBS@ @OMNIDYNAMIC_LIBS@ @LOG4CXX_LIBS@
+Cflags: -I${includedir} -I${extraincludedir} -I${idldir} @OMNICOS_CFLAGS@ @OMNIORB_CFLAGS@ @OMNIDYNAMIC_CFLAGS@ @LOG4CXX_CFLAGS@ @HAVE_LOG4CXX@ @ENABLE_EVENTS_FLAGS@
-- 
